client/src/components/Voter/VoterContainer.jsx

```diff
@@ -1,11 +1,52 @@
+import useEth from "../../contexts/EthContext/useEth";
+import { useState } from "react";
+
+
 const VoterContainer = () => {
+  const { state: { contract, accounts } } = useEth();
+  const [inputValue, setInputValue] = useState("");
```

Comment:
@rembeuch 
Tu devrais renommer comme cela pour que ce soit plus explicite ;-)
```suggestion
  const [voterAddress, setVoterAddress] = useState("");
```